docs/_code/movies/Person.php

```diff
@@ -104,4 +104,4 @@ public function removeMovie(Movie $movie)
             $this->movies->removeElement($movie);
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
I assume GitHub did this automatically, I did not intentionally add the closing brace.